### ISSUE TITLE
Wrong module path?

### ISSUE THREAD

**Issue by NonPolynomial** [NoDivide/astrum#64]
#### Tell us about the issue you are experiencing

If i try to `astrum init ./patterns/` (or any other path) i get the following error:

```
Error: Error: ENOENT: no such file or directory, lstat 'C:\Users\NonPolynomial\AppData\Roaming\npm\node_modules\astrum\_template'
```

It seems that astrum tries to get the template files from my appdata folder, but node and npm are installed in `C:\Node`
#### What behaviour would you expect?

Astrum should look for the template files in the folder node / npm / astrum was installed in.
#### How can this issue be replicated?

`astrum init ./patterns/`
#### Your environment

Platform: PC
OS Version: Windows 8.1
Node Version: 6.3.0
NPM Version: 3.10.5
Astrum Version: 1.6.3

---

Editted: fixed typo

**Comment by NonPolynomial** [comment]
i could solve this with following changes on manager/utils.js:

```js
var fs = require('fs-extra'),
    chalk = require('chalk'),
    inquirer = require('inquirer'),
    dir = require('global-modules'),
    mkdirp = require('mkdirp'),
    isWindows = require('is-windows'),
    pjson = require('../package.json'),
    path = require('path');

module.exports = {

    // resolve module path with process.argv
    module_path: path.normalize(`${path.dirname(process.argv[1])}/../`),

    /* ... */
}
```

**Comment by RyanHavoc** [comment]
@NonPolynomial I'm assuming then that you install your global node modules into a non-default location on Windows? I was expecting the global-modules dependency to resolve the correct path in that case...

Thanks for the feedback on what change could be made. I'll test it out, I just need to ensure it sets the correct path in all instances. Mac and Windows alike.

**Comment by NonPolynomial** [comment]
hey @RyanHavoc ,

i expected `global-modules` to do this, too. :c
Shouldn't `process.argv` always do the trick here?
I thought, it wouldn't be bound to any OS.

Further explanation:
i didn't "install" node. i just downloaded the node.exe and the npm tarball and unpacked it, and set the environment variable to node myself, because thats what the installer just do.

**Comment by RyanHavoc** [comment]
@NonPolynomial It should do the trick but then again I wouldn't have expected it to break in your case in the first place. Testing required!

**Comment by NonPolynomial** [comment]
Do you plan to include a bugfix for that in any version? @RyanHavoc

**Comment by RyanHavoc** [comment]
@NonPolynomial Could you try out the `release/2.0.0` branch and see if your problem has been resolved in that release?

**Comment by NonPolynomial** [comment]
I'm afraid, it's not :/

`_this.module_path` targets the wrong path in appdata in [`manager/utils.js`](https://github.com/NoDivide/astrum/blob/release/2.0.0/manager/utils.js#L52).

**Comment by NonPolynomial** [comment]
@RyanHavoc would you prefer, when I take a fork and apply the changes to `master` or to the `release/2.0.0` branch?

**Comment by RyanHavoc** [comment]
@NonPolynomial 'master' please. It's going to be a little while before v2.0.0 is ready for release.

**Comment by NonPolynomial** [comment]
@RyanHavoc fixed with the merge #96 -> Release v1.9.5
